.github/workflows/smoke.yml

```diff
@@ -42,6 +42,44 @@ jobs:
       - name: Build Android debug
         run: flutter build appbundle --debug
 
+      - name: Setup bundletool
+        if: github.ref == 'refs/heads/main'
+        uses: amyu/setup-bundletool@f7a6fdd8e04bb23d2fdf3c2f60c9257a6298a40a
+      - name: Install the google play key material
+        if: github.ref == 'refs/heads/main'
+        env:
+          GOOGLE_PLAY_API_JWT_BASE64: ${{ secrets.GOOGLE_PLAY_API_JWT_BASE64 }}
+          GOOGLE_PLAY_KEYSTORE_BASE64: ${{ secrets.GOOGLE_PLAY_KEYSTORE_BASE64 }}
+        run: |
+          GOOGLE_PLAY_API_JWT_PATH="$RUNNER_TEMP/gp_api.json"
+          echo "GOOGLE_PLAY_API_JWT_PATH=$GOOGLE_PLAY_API_JWT_PATH" >> $GITHUB_ENV
+          echo -n "$GOOGLE_PLAY_API_JWT_BASE64" | base64 --decode --output "$GOOGLE_PLAY_API_JWT_PATH"
+
+          GOOGLE_PLAY_KEYSTORE_PATH="$RUNNER_TEMP/gp_signing.jks"
+          echo "GOOGLE_PLAY_KEYSTORE_PATH=$GOOGLE_PLAY_KEYSTORE_PATH" >> $GITHUB_ENV
+          echo -n "$GOOGLE_PLAY_KEYSTORE_BASE64" | base64 --decode --output "$GOOGLE_PLAY_KEYSTORE_PATH"
+      - name: Generate debug apk
+        if: github.ref == 'refs/heads/main'
+        env:
+          TOKEN: ${{ secrets.MACHINE_USER_PAT }}
+          GOOGLE_PLAY_KEYSTORE_PASSWORD: ${{ secrets.GOOGLE_PLAY_KEYSTORE_PASSWORD }}
+        run: |
+          bundletool build-apks \
```

Comment:
Why not `flutter build apk`?

Reply:
I thought maybe it's useful to still create the app bundle as a smoke test, since that is closer to how we create our releases.